src/bootstrap/bootstrap.py

```diff
@@ -934,19 +934,20 @@ def main():
             "      See src/bootstrap/README.md for help with common commands."
         )
 
-    exit_code = 0
     try:
         bootstrap(args)
+        if not help_triggered:
+            print("Build completed successfully in", format_build_time(time() - start_time))
     except (SystemExit, KeyboardInterrupt) as error:
         if hasattr(error, 'code') and isinstance(error.code, int):
             exit_code = error.code
         else:
             exit_code = 1
             print(error)
 
-    if not help_triggered:
-        print("Build completed successfully in", format_build_time(time() - start_time))
```

Comment:
can't you just do `code == 0 ? "successfully" : "unsuccessfully"` so we can combine these codepaths (not literally using a ternary, but conceptually)

Reply:
you mean like this?

```python
success_word = "successfully"
try:
    bootstrap(args)
except (SystemExit, KeyboardInterrupt) as error:
    # ...
    success_word = "unsuccessfully"
```

Reply:
This might just be me being too nit-picky, but "Build completed unsuccessfully" - does that make sense? Some of the places we do some sanity checks before even running the code, so it happens that we exit with error premature to actually "running" something.

How about "Build failed" or "Build was unsuccessful"?

Reply:
let's please keep the original message until we fix the regression, we can bikeshed the message later.